StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/CustomerSheet/CustomerSheet.swift

```diff
@@ -212,6 +221,15 @@ public class CustomerSheet {
         }
         return nil
     }
+    
+    private func createInitEvent() -> STPAnalyticEvent {
```

Comment:
Super minor (not blocking): This could be a computed property instead of a function:
```suggestion
    private var initEvent: STPAnalyticEvent {
```